Sources/MarathonCore/Command.swift

```diff
@@ -90,6 +93,8 @@ extension Command {
             return ""
         case .help:
             return ""
+        case .export:
+            return "<script-path> [<export-path>] [--force]"
```

Comment:
Ideally the order shouldn't matter, but I think it's out of scope for this PR. Better make it behave like the other commands are, which it is now doing. So 👍